### PR TITLE
Relocate functions in items.php into several classes

### PR DESCRIPTION
**Pull request by annando**
This is some WIP, but I would like to do the PR anyway to not cause too much interference with other PRs.

I added several hints where to place the functions in the future - for some I'm still unsure.

### CODE REVIEW

File: src/Model/Item.php [hunk 1]
```diff
@@ -290,4 +291,135 @@ public static function addShadowPost($itemid)
...
+	}
+
+	/**
+	 * @brief Creates an unique guid out of a given uri
+	 *
+	 * @param string $uri uri of an item entry
+	 * @param string $host (Optional) hostname for the GUID prefix
+	 * @return string unique guid
+	 */
+	public static function GuidFromUri($uri, $host = "")
```
Comment (MrPetovan) [inline src/Model/Item.php]: Standards: Please fix the capitalization of this method's name (no capital first letter)

File: src/Model/Conversation.php [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+class Conversation
+{
+	/**
+	 * @brief Store the conversation data
+	 *
+	 * @param array $arr Item array with conversation data
+	 * @return array Item array with removed conversation data
+	 */
+	public static function insert($arr) {
```
Comment (MrPetovan) [inline src/Model/Conversation.php]: We should agree on a common naming scheme for Model classes, currently we use `create`, `add` and `insert` for the same operation.

Comment (annando) [reply src/Model/Conversation.php]: I would like to orientate after the SQL commands, mean: "insert", "update" and "delete". 

BTW: You forgot ```store``` (as seen in Model/Photo.php) :-)

Comment (MrPetovan) [reply src/Model/Conversation.php]: But `store` is a combination of `insert/update`, so not strictly the same as all the above!

File: src/Model/Item.php (hunk 1 above)
Comment (annando) [reply src/Model/Item.php]: Sorry, it's just like we do on work ... (Is now fixed)

File: src/Model/Conversation.php (hunk 2 above)
Comment (annando) [reply src/Model/Conversation.php]: Okay. What would you suggest?

Comment (MrPetovan) [reply src/Model/Conversation.php]: https://friendica.mrpetovan.com/display/735a2029915a63dda2c8b7b601667320

**Comment by annando** [comment]
These comments remind us that we have to decide where to move it. Since this file won't be there for a long time, we shouldn't be too picky about this.

**Comment by MrPetovan** [comment]
But we would have had to decide where to move it without the comment anyway 😉

**Pull request merged**